src/components/shared/PageContainer.tsx

```diff
@@ -9,8 +20,48 @@ const PageContainerPropTypes = {
 };
 type PageContainerProp = PropTypes.InferProps<typeof PageContainerPropTypes>;
 
+const selectors = {
+    hideNotification: (state: NotificationState) => state.hideNotification,
+    notification: (state: NotificationState) => state.notification,
+    updateNotificationHistory: (state: NotificationState) =>
+        state.updateNotificationHistory,
+};
+
 function PageContainer(props: PageContainerProp) {
+    const alertTransitionRef = useRef<any>(null);
```

Comment:
The attributed type will be narrowed in a subsequent PR. Since I was experimenting with different transition components, the `any` type was temporarily employed to minimize the code changes required for each transition.